### PR TITLE
[NU-1932] Big decimal uses default scale

### PR DESCRIPTION
**Pull request by paw787878**
## Checklist before merge
- [x] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [x] Code is cleaned from temporary changes and commented out lines
- [x] Parts of the code that are not easy to understand are documented in the code
- [x] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [x] Verify that PR will be squashed during merge

### CODE REVIEW

File: utils/utils/src/main/java/pl/touk/nussknacker/springframework/util/NumberUtilsConsts.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package pl.touk.nussknacker.springframework.util;
+
+public class NumberUtilsConsts {
```
Comment (arkadius) [inline utils/utils/src/main/java/pl/touk/nussknacker/springframework/util/NumberUtilsConsts.java]: Let's rename this class to some name that would explain why we do this trick e.g. `BigDecimalScaleEnsurer`, move all setScale invocations and write a notice that this is work around for spel's behaviour with an explanation how spel behaves. Basically let's move our online conversion to comment :)

File: scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/extension/Conversion.scala [hunk 2]
```diff
@@ -68,11 +69,15 @@ object ToBigDecimalConversion extends ToNumericConversion[JBigDecimal] {
 
   override def convertEither(value: Any): Either[Throwable, JBigDecimal] =
     value match {
-      case v: JBigDecimal => Right(v)
-      case v: JBigInteger => Right(new JBigDecimal(v))
+      case v: JBigDecimal => Right(v.setScale(Math.max(v.scale(), NumberUtilsConsts.DEFAULT_BIG_DECIMAL_SCALE)))
```
Comment (lciolecki) [inline scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/extension/Conversion.scala]: Let's extract:

```
v.setScale(Math.max(v.scale(), NumberUtilsConsts.DEFAULT_BIG_DECIMAL_SCALE))
```

to separated method.

File: utils/utils/src/main/java/pl/touk/nussknacker/springframework/util/NumberUtilsConsts.java (hunk 1 above)
Comment (paw787878) [reply utils/utils/src/main/java/pl/touk/nussknacker/springframework/util/NumberUtilsConsts.java]: done

File: scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/extension/Conversion.scala (hunk 2 above)
Comment (paw787878) [reply scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/extension/Conversion.scala]: done

**Review by arkadius — COMMENTED** [review]
Please add changelog entry and fix tests

**Review by arkadius — APPROVED** [review]

**Pull request merged**